dm/pkg/conn/utils.go

```diff
@@ -234,9 +234,9 @@ func GetPosAndGs(ctx *tcontext.Context, db *BaseDB, flavor string) (
 	}
 	binlogPos = gmysql.Position{
 		Name: binlogName,
-		Pos:  pos,
+		Pos:  uint32(pos),
 	}
-
+	ctx.L().Warn("Convert the pos type returned by GetMasterStatus to uint32 to fit binlog position")
```

Comment:
seems we can only log a warning when the value is larger than max value of uint32 of error here 🤔 I prefer warning because it may not in a critical path